app/frontend/src/components/designer/profile/FormMinistryProfile.vue

```diff
@@ -19,9 +19,9 @@ const form = computed(() => formStore.form);
 const isRTL = computed(() => formStore.isRTL);
 
 const MinistryList = computed(() => {
-  return Ministries.map((ministry) => ({
-    id: ministry.id,
-    text: t(`trans.ministries.${ministry.id}`),
+  return Ministries.map((Organization) => ({
```

Comment:
this change doesnt have any effect though since the Organization or ministry is just a temp variable